lib/errors/error-collector.js

```diff
@@ -288,10 +360,20 @@ class ErrorCollector {
 
     if (exception.error) {
       logger.trace(exception.error, 'Got exception to trace:')
+    } else {
+      logger.trace(transaction, 'Got transaction error to trace:')
```

Comment:
can you explain this? i don't see it in the old but this stuff was very involved before

Reply:
ah ok, no, it's fine